### ISSUE TITLE
Render skate parks differently then other pitches

### ISSUE THREAD

**Issue by Adamant36** [gravitystorm/openstreetmap-carto#4148]
I'd like to suggest either rendering skate parks in a different color then other pitches or rendering them with an icon. There are several reasons why I think they should have a unique rendering. 

1. They are kind of their own "facility", similar to sports centers, dog parks, or playgrounds. So, it would be helpful if it was easy to tell them apart from something that isn't like a tennis court.

2. Also like how dog parks are a separate thing from parks more generally, there is a specific group of people that are interested in skate parks as their own object outside of other sports pitches. 

2. If we went with an icon it would not clutter the rendering like icons for other pitches do, because there is not a bunch of small skate parks lined up next to each other like there is for other pitch types. 

3. One possible option would be to render them similar to dog parks. With a slightly different fill color to other pitches (people don't really consider them pitches anyway) and an icon pattern. 

As a side, currently almost 800 skate parks are tagged with a leisure tag other then leisure=pitch and a couple of thousand are currently tagged with some variation of name=skatepark. Just going on the bad tagging alone there's clearly a need for unique rendering. It would also appeal to "skaters" though.

**Comment by imagico** [comment]
Could you please specify what a skate park is and how it is tagged in OSM?

**Comment by Adamant36** [comment]
My bad. Its a place where you ride a skateboard. Or leisure=pitch + sport=skateboard in OSM.

**Comment by polarbearing** [comment]
> dog parks... icon pattern

The pattern in dog parks was a bad idea, I'd like to see that reduced to a single icon. Thus no more such patterns. 

No objections against an icon for the skate park, however that opens the discussions about sports icons in general.

**Comment by Adamant36** [comment]
> The pattern in dog parks was a bad idea, I'd like to see that reduced to a single icon.

I mostly agree with that. 

> however that opens the discussions about sports icons in general.

I don't see why. There's plenty of tags where there is only rendering for a small range of keys and no one is clamoring for the rest to be rendered, or even if they are it's not like it matters or stopped you guys from implementing it.

**Comment by polarbearing** [comment]
As said, I do not mean to object the skate icon. I just assume that when you start rendering a 2nd level tag that the demand increases to do the same for tags on the same level.

**Comment by jeisenbe** [comment]
This would make sense in context with rendering other types of `sport` tags on `leisure=pitch` features.

**Comment by jeisenbe** [comment]
Related to #844 or #1126

**Comment by jeisenbe** [comment]
It appears that this issue cannot be solved alone, rather it is a part of the open issue #844 which includes other sports, so I will close it as a duplicate. 

However, please note that I am still personally interested in seeing a new PR which would show a feasible way to distinguish pitches by sport.

**Comment by Adamant36** [comment]
> It appears that this issue cannot be solved alone, rather it is a part of the open issue #844 which includes other sports, so I will close it as a duplicate.

But I opened it as a separate issue for the reasons I stated in my first commented. Skate parks being more like their own facilities and whatnot. No one here has said why they shouldn't be considered separately from something small without various other components in it like a tennis court. I don't think taking an all or nothing approach to this is the way to go anymore then it would have been to take one originally with the landuse or leisure tags. Not all pitches aren't the same and they shouldn't be treated that way. Especially with skate parks. Which most people don't consider traditional sports pitches. Which is they are called "parks."
